elements/urth-viz-table/urth-viz-table.html

```diff
@@ -235,7 +235,7 @@
                 // parse columns using data's first row
                 // in case user provides missing column headers
                 var datarows = this.datarows;
-                console.assert(datarows);
+                this._assert(datarows);
```

Comment:
ha! Thanks... this was giving me an error in testing... I did a bulk find/replace.